docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -82,6 +84,38 @@ This video explains how to add data sources, including Loki, Tempo, and Mimir, t
 
 {{< youtube id="cqHO0oYW6Ic" start="298" >}}
 
+## Streaming
+
+<!-- Streaming toggle will be deprecated in Grafana 11.2 and removed in 11.3. -->
+
+Streaming provides immediate visibility of incoming traces on the results table for a TraceQL query.
+
+The Streaming section is available in the Tempo data source settings for Grafana Cloud.
+This capability requires Tempo 2.2 and onwards.
+
+### Requirements
+
+To use this feature, you need to:
+
+- Use Tempo version 2.2 or newer. Grafana Cloud uses has the latest version of Tempo.
+- For self-managed Tempo instances: If your Tempo instance is behind a load balancer or proxy that doesn't supporting gRPC or HTTP2, streaming may not work and should be disabled.
+
+For streaming to work for a particular Tempo data source, set both `traceQLStreaming` feature toggle to true and **Streaming** enabled in the the data source to allow streaming.
+
+If the data source allows streaming but `traceQLStreaming=false`, no streaming will occur.
+
+If the data source has streaming disabled and `traceQLStreaming=true`, no streaming wil happen for that data source.
+
+### Activate streaming
```

Comment:
This section seems sort of redundant when compared to what is already above so perhaps not needed? Will defer to you however!

Reply:
I think I might restructure the information here. Some of the content we have in Requirements should be in the procedural part.

Reply:
Restructured.